### DISCUSSION TITLE
Android (Material You palette)

### DISCUSSION THREAD

**Discussion by PoorPocketsMcNewHold** [catppuccin/catppuccin#2130] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

https://repainter.app/
https://play.google.com/store/apps/details?id=dev.kdrag0n.dyntheme

While you could force the Monet engine to create a fine-enough Material You palette with a catppucinnified wallpaper, you don't have any specific personalizations over it, and it restrict you to use a specific wallpaper for that usage.
However, for the few who can make this one work, Repainter can actually allow to create your own palette, and in fact, even share those configurations !

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by justTOBBI** [comment]
~Interested in creating said theme~

**Comment by Aleksanaa** [comment]
Really looks promising.

Edit: I hope there could be an alternative. Just don't want to guarantee root (or wireless debug) to a proprietary app.

**Reply by PoorPocketsMcNewHold** [reply]
Sadly, haven't found any FOSS alternatives so far, but interested in one.
I did found an alternative non-root way to customize the Money colors via a Live-Wallpaper.
https://play.google.com/store/apps/details?id=com.lb.lwp_plus

There's an another proprietary app for such thing i recently found. https://play.google.com/store/apps/details?id=com.pranavpandey.theme

**Reply by Covkie** [reply]
Colorblendr seems to be the best solution here. https://f-droid.org/packages/com.drdisagree.colorblendr/

**Comment by MJ-asa** [comment]
This can also work with the root app Iconify

**Reply by MJ-asa** [reply]
Iconify today removed their monet engine features, saying the best way to do it would be Colorblendr

**Comment by Ermageeerd** [comment]
I am currently doing this as well as a Heliboard theme. Link is at https://github.com/Ermageeerd/Android-Catppucin.
